### ISSUE TITLE
Add a section for articles about the Solus organization

### ISSUE THREAD

**Issue by EbonJaeger** [getsolus/help-center-docs#503]
My inspiration for this comes largely (okay, entirely) from the equivalent section of the Buddies of Budgie documentation site, [here](https://docs.buddiesofbudgie.org/organization/intro). The pages they have in that section would also fit for Solus. Thus, I propose the following:

1. Add a new main documentation section
  - Add a simple landing page
  - Add a page describing our values as a Linux distribution
  - Add a page with an overview of the different Solus teams
    - This page will likely be the trickiest to implement; the Budgie site uses a handful of custom components for displaying teams and people
    - While it wouldn't look as cool, it could also be a simpler text page outlining the different teams
  - Add a page showing people how they can get involved with the project
    - Filing issues on our issue tracker
    - Updating and maintaining packages
    - Contributing translations to Solus software projects
    - Financially via Open Collective
2. Rename the existing Packaging section to Contributing
  - This doesn't necessarily need to be done after

**Comment by TraceyC77** [comment]
We now have a translation instructions doc, but the only thing it says is to join our Matrix rooms, and has stuff specific to the Mate Transition Tool

**Comment by EbonJaeger** [comment]
Now that our organizational stuff is on the main site, this should be closed out. If there is still an issue with translation documentation, either here or on the main site, a separate issue should be opened for that.

**Comment by TraceyC77** [comment]
OK, what is this waiting on before it can be closed?

**Comment by EbonJaeger** [comment]
In my opinion, nothing.
